docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3"
+
+services:
+
+  backend:
+    image: rubix982/jina-backend:v1.0.0    
```

Comment:
Shall we have an organization on Docker Hub, where to host the thing? This is good for now, but it might change in the future (when we have a definitive name and the thing running, I'd say).

Reply:
Yes, I was unclear where you would want to put the images on, so I put it on my personal account for now. But you can change it to anything you like. The format is `[username]/[repository_name]:[tag]`, where `username` and `repository_name` is from Docker  Hub.